### PR TITLE
Improve test coverage for Grid layers and utils

### PR DESCRIPTION
**Pull request by 1chandu**
For #

#### Background
Improve test coverage for Grid layers and utils

#### Change List
- Improve test coverage for Grid layers and utils

### CODE REVIEW

File: test/modules/aggregation-layers/index.js [hunk 1]
```diff
@@ -22,10 +22,12 @@ import './contour-layer/contour-layer.spec';
...
 import './gpu-grid-layer/gpu-grid-layer.spec';
+import './gpu-grid-layer/gpu-grid-cell-layer.spec';
 import './cpu-grid-layer/cpu-grid-layer.spec';
 import './grid-aggregator.spec';
 import './hexagon-layer.spec';
 import './hexagon-aggregator.spec';
 import './grid-layer.spec';
 import './screen-grid-layer.spec';
 import './utils/scale-utils.spec';
+import './utils/aggregation-operation-utils.spec';
```
Comment (Pessimistress) [inline test/modules/aggregation-layers/index.js]: This needs to be removed from `test/browser.js`

File: modules/aggregation-layers/src/utils/color-utils.js [hunk 2]
```diff
@@ -29,6 +29,9 @@ export const defaultColorRange = [
 
 // Converts a colorRange array to a flat array with 4 components per color
 export function colorRangeToFlatArray(colorRange, ArrayType, defaultValue) {
+  if (colorRange.constructor === ArrayType) {
```
Comment (Pessimistress) [inline modules/aggregation-layers/src/utils/color-utils.js]: If the intention is to support a flat array from the user, we should relax this check to be `Arraybuffer.isView` or even `Numer.isFinite(colorRange[0])`

File: test/modules/aggregation-layers/index.js (hunk 1 above)
Comment (1chandu) [reply test/modules/aggregation-layers/index.js]: Any specific reason? Its testing pure JS code only.

Comment (Pessimistress) [reply test/modules/aggregation-layers/index.js]: `test/browser.js` already imports `modules/index.js` so this ends up being imported twice.

**Pull request merged**